### PR TITLE
Fixing #7615 : Documented required things

### PR DESCRIPTION
**Pull request by Yathartha22**
This PR is regarding [issue #7615](https://github.com/sympy/sympy/issues/7615).

### CODE REVIEW

File: .profile [hunk 1]
```diff
@@ -0,0 +1 @@
+export PATH="$PATH:/c/Python27"
```
Comment (asmeurer) [inline .profile]: Remove this file.

File: sympy/solvers/solvers.py [hunk 2]
```diff
@@ -451,6 +451,21 @@ def solve(f, *symbols, **flags):
 
         >>> solve(x < 3)
         And(-oo < x, x < 3)
+    
+    * to solve set of equations when **flag set to default values
```
Comment (asmeurer) [inline sympy/solvers/solvers.py]: This is unnecessary. The other examples already show this.

File: sympy/solvers/solvers.py [hunk 3]
```diff
@@ -459,11 +474,22 @@ def solve(f, *symbols, **flags):
         >>> solve([x - 3, y - 1], dict=True)
         [{x: 3, y: 1}]
 
+     
+    ** to get only desired value value if dict=True
```
Comment (asmeurer) [inline sympy/solvers/solvers.py]: There's no need to duplicate the example here. Just add an example to the above of accessing the solution. Something like

```
>>> sol = solve([x - 3, y - 1], dict=True)
>>> sol
[{x: 3, y: 1}]
>>> sol[0][x]
3
```

**Review by asmeurer — CHANGES_REQUESTED** [review]

File: sympy/solvers/solveset.py [hunk 4]
```diff
@@ -1270,7 +1270,7 @@ def linsolve(system, *symbols):
     else:
         raise ValueError("Invalid arguments")
 
-    # Solve using Gauss-Jordan elimination
+    #Solve using Gauss-Jordan elimination
```
Comment (asmeurer) [inline sympy/solvers/solveset.py]: Don't remove the spaces after the comment symbol.

**Comment by asmeurer** [comment]
Be sure to check the code quality tests (`./bin/test quality`) before commiting.

**Comment by Yathartha22** [comment]
@asmeurer I have made the changes. Please have a look

**Review by asmeurer — APPROVED** [review]

**Comment by asmeurer** [comment]
Please try to use a more descriptive pull request title and more descriptive commit messages in the future. See https://github.com/sympy/sympy/wiki/Development-workflow#writing-commit-messages.

**Comment by Yathartha22** [comment]
Thanks @asmeurer this was my first contribution. I will definitely make more of better contributions, and surely keep in mind what you said :)

**Pull request merged**